frontend/packages/ceph-storage-plugin/src/components/ocs-install/install-wizard/_configure.scss

```diff
@@ -13,4 +13,7 @@
 
 .ceph__multus-dropdown-button {
   min-width: 18em;
+  .text-muted {
+    display: none !important;
+  }
```

Comment:
This will only hide for ceph.

We want a BEM class name specific to the resource dropdown like `co-resource-dropdown__namespace` and then hide that text in the button. We shouldn't be keying on a generic class like `text-muted`, but we shouldn't scope it just to ceph either.